public/locales/en.json

```diff
@@ -118,7 +118,7 @@
   "migrateInitiated": "Migrate initiated",
   "migrationAmount": "Migration Amount",
   "migrationExplain": "This pool is outdated. We recommend migrating to the new gas optimized version to earn better returns.",
-  "minRemaining": "min remaining",
+  "minRemaining": "mininutes left",
```

Comment:
Two issues:
- we already have a key for `minutesLeft` below
- "minutes" is misspelled here